src/test/java/com/savvato/tribeapp/services/PhraseServiceImplTest.java

```diff
@@ -60,6 +60,12 @@ public PhraseService phraseService() {
     @MockBean
     UserPhraseRepository userPhraseRepository;
 
+    @MockBean
+    ToBeReviewedRepository toBeReviewedRepository;
+
+    @MockBean
+    ReviewSubmittingUserRepository reviewSubmittingUserRepository;
+
```

Comment:
We need a test for the new code..

Reply:
Added two tests for applyPhraseToUser:
Test that reviewSubmittingUserRepository is called when the phrase already exists in to_be_reviewed
Test that reviewSubmittingUserRepository is called when the phrase does not exist in to_be_reviewed